src/lightning_app/cli/lightning_cli.py

```diff
@@ -55,7 +55,7 @@ def _main():
 
 @_main.command()
 def login():
-    """Log in to your Lightning.ai account."""
+    """Log into your lightning.ai account."""
```

Comment:
https://english.stackexchange.com/questions/5302/log-in-to-or-log-into-or-login-to

@Felonious-Spellfire

Reply:
@awaelchli (CC @RobertLaurella) here's the debate:

Log in, log on, sign in, sign on

https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/sign-in-sign-out
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/l/log-on-log-off
https://support.apple.com/guide/applestyleguide/l-apsg087a9dba/1.0/web/1.0#apdca21e0f4bea34
https://support.apple.com/guide/applestyleguide/s-apsge70df12b/1.0/web/1.0#apd2ad58521c67c4

I honestly don't care which one we decide on, as long as once we make the decision it is always used. I've been slowly adding things to this: https://www.notion.so/lightningai/Terminology-ad3504ffae2c4d3e80156345d11af63e so we can track what "official" style we want to follow.

Reply:
```suggestion
    """Log in to your lightning.ai account."""
```

Reply:
Hey team - love this topic lol - this would be "Log in to [x]" because "into" is a preposition, in other words: "I placed all of my files into the storage provided by my Lightning account" because the files are being placed somewhere that the preposition describes. "Log into" would imply that you're logging [in an area] because into is a preposition, but in this case the verb is "to log in", not simply "to log".